crates/ark/src/lsp/statement_range.rs

```diff
@@ -1124,46 +1112,47 @@ if (a > b)
 ",
         );
 
+        // TODO: This should run the whole if statement because there are no braces
         statement_range_test(
             "
 if (a > b)
-  <<@1 + 1>> else if (b > c)
-  2 + 2 else 4 + 4
+  <<@1 + 1 else if (b > c)
+  2 + 2 else 4 + 4>>
 ",
         );
 
-        // TODO: I'm not exactly sure what this should run, but this seems strange
+        // TODO: This should run the whole if statement because there are no braces
         statement_range_test(
             "
 if (a > b)
-  <<1 + 1>> else if @(b > c)
-  2 + 2 else 4 + 4
+  <<1 + 1 else if @(b > c)
+  2 + 2 else 4 + 4>>
```

Comment:
It does make these _extremely_ weird edge cases a little "worse" because previously we at least selected a complete expression before, but as you can see from the previous TODO we didn't really know the right thing to return here.

We now thing the right thing to return is unambiguously the entire top level `if` statement, so I've updated the TODO and I'm not really caring too much about the current result.

I don't anticipate it affecting anyone in practice really